tests/Doctrine/Migrations/Tests/Tools/Console/Command/DoctrineCommandTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Migrations\Tests\Tools\Console\Command;
+
+use Doctrine\Migrations\DependencyFactory;
+use Doctrine\Migrations\Tests\MigrationTestCase;
+use Doctrine\Migrations\Tools\Console\Command\DoctrineCommand;
+use Symfony\Component\Console\Tester\CommandTester;
+
+class DoctrineCommandTest extends MigrationTestCase
+{
+    public function testCommandFreezes() : void
+    {
+        $dependencyFactory = $this->getMockBuilder(DependencyFactory::class)
+            ->disableOriginalConstructor()
+            ->setMethods(['freeze'])
+            ->getMock();
+
+        $dependencyFactory
+            ->expects($this->once())
```

Comment:
This should fix the build:
```suggestion
            ->expects(self::once())
```